worlds/messenger/client_setup.py

```diff
@@ -56,27 +60,34 @@ def install_courier() -> None:
         if not is_windows:
             mono_exe = which("mono")
             if not mono_exe:
-                # steam deck support but doesn't currently work
-                messagebox("Failure", "Failed to install Courier", True)
-                raise RuntimeError("Failed to install Courier")
-                # # download and use mono kickstart
-                # # this allows steam deck support
-                # mono_kick_url = "https://github.com/flibitijibibo/MonoKickstart/archive/refs/heads/master.zip"
-                # target = os.path.join(folder, "monoKickstart")
-                # os.makedirs(target, exist_ok=True)
-                # with urllib.request.urlopen(mono_kick_url) as download:
-                #     with ZipFile(io.BytesIO(download.read()), "r") as zf:
-                #         for member in zf.infolist():
-                #             zf.extract(member, path=target)
-                # installer = subprocess.Popen([os.path.join(target, "precompiled"),
-                #                               os.path.join(folder, "MiniInstaller.exe")], shell=False)
-                # os.remove(target)
+                # download and use mono kickstart
+                # this allows steam deck support
+                mono_kick_url = "https://github.com/flibitijibibo/MonoKickstart/archive/refs/heads/master.zip"
```

Comment:
This seems dangerous to link to a head.
I think you can get the same thing more safely with this:
`https://github.com/flibitijibibo/MonoKickstart/archive/716f0a2bd5d75138969090494a76328f39a6dd78.zip`

Reply:
i was fine with doing it that way because it's been archived for so long i highly doubt it'll actually get an update but yeah better safe than sorry